terraform/environments/electronic-monitoring-data/platform_versions.tf

```diff
@@ -8,6 +8,12 @@ terraform {
       version = "~> 3.0"
       source  = "hashicorp/http"
     }
+  null = {
```

Comment:
was this to do with the weird error you were seeing with terraform?

Reply:
this fixed the error